js/sliderView.js

```diff
@@ -180,107 +177,105 @@ define([
-      var $markers = this.$('.js-slider-scale-notch-container').empty();
+    showScale() {
+      const $markers = this.$('.js-slider-scale-notch-container').empty();
 
       if (this.model.get('_showScale') === false) {
-        $markers.eq(0).css({display: 'none'});
-        this.$('.js-slider-number').css(
-          this.model.get('_showScaleIndicator') ? {visibility: 'hidden'} : {display: 'none'}
+        $markers.eq(0).addClass('u-display-none');
+        this.$('.js-slider-number').addClass(
+          this.model.get('_showScaleIndicator') ? 'visibility-hidden' : 'u-display-none'
         );
         return;
       }
 
-      var $scaler = this.$('.js-slider-scale');
-      for (var i = 1, count = this.model.get('_items').length - 1; i < count; i++) {
-        $markers.append('<div class="slider__scale-notch" style="left: ' + this.mapIndexToPixels(i, $scaler) + 'px">');
+      const $scaler = this.$('.js-slider-scale');
+      for (let i = 1, count = this.model.get('_items').length - 1; i < count; i++) {
+        $markers.append(`<div class="slider__scale-notch" style="left: ${this.mapIndexToPixels(i, $scaler)}px">`);
       }
       // Do we show scale numbers
       this.showScaleNumbers();
-    },
+    }
 
-    showScaleNumbers: function () {
-      var $scaler = this.$('.js-slider-scale');
-      var $numbers = this.$('.js-slider-number');
+    showScaleNumbers() {
+      const $scaler = this.$('.js-slider-scale');
+      const $numbers = this.$('.js-slider-number');
 
       if (this.model.get('_showScaleNumbers') === false) {
-        $numbers.css('display', 'none');
+        $numbers.addClass('u-display-none');
         return;
       }
 
-      var scaleWidth = $scaler.width();
-      this.model.get('_items').forEach(function(item, index) {
-        var $number = $numbers.eq(index);
-        var newLeft = Math.round($number.data('normalisedPosition') * scaleWidth);
-        $number.css({left: newLeft});
+      const scaleWidth = $scaler.width();
+      this.model.get('_items').forEach((item, index) => {
+        const $number = $numbers.eq(index);
+        const newLeft = Math.round($number.data('normalisedPosition') * scaleWidth);
+        $number.css({ left: newLeft });
       });
-    },
+    }
 
     //Labels are enabled in slider.hbs. Here we manage their containing div.
-    showLabels: function () {
-      if (!this.model.get('labelStart') && !this.model.get('labelEnd')) {
-        this.$('.js-slider-label-container').eq(0).css({display: 'none'});
-      }
-    },
-
-    remapSliderBar: function() {
-      var $scaler = this.$('.js-slider-scale');
-      var currentIndex = this.getIndexFromValue(this.model.get('_selectedItem').value);
-      var left = this.mapIndexToPixels(currentIndex, $scaler);
-      this.$('.slider-handle').css({left: left + 'px'});
-      this.$('.js-slider-number-selection').css({left: left + 'px'});
+    showLabels() {
+      if (this.model.get('labelStart') && this.model.get('labelEnd')) return;
```

Comment:
```suggestion
      if (this.model.get('labelStart') || this.model.get('labelEnd')) return;
```
You need to invert the operators as well